### ISSUE TITLE
Valid URLs returning "about:blank"

### ISSUE THREAD

**Issue by ccrawford4** [codesplinta/URISanity#1]
Here is an example of a valid URL that I am trying to test:
https://www.customer-site.com/hubfs/3.%20Client%20Documents/0Benefits%20Guide.pdf

*changed for privacy sake

But despite trying to use the least restrictive parameters here:
```typescript
const sanitizedLink = URISanity.vet(url, {
              allowScriptOrDataURI: true,
              allowFileSystemURI: true,
              allowCommsAppURI: true,
              allowDBConnectionStringURI: true,
              allowBrowserSpecificURI: true,
              allowWebTransportURI: true,
              allowServiceAPIURI: true,
            });
console.log(sanitizedLink);
```

The URL returned is still "about:blank". Is there a reason that this URL is being flagged as suspicious? In short I want my customer to be able to use any valid URLs that don't contain malicious scripts embedded within them.

**Comment by isocroft** [comment]
Hello @ccrawford4 , I have investigated this and a fix should be out by the second week of October as i am quite busy at this moment.

This is due to a false negative matching the `%20` encoded character before a non-alphanumeric character `.`.

Thank you for sending in this issue.

**Comment by isocroft** [comment]
Hello @ccrawford4 , Please download the latest version of URISanity (v0.1.5) for the fix

**Comment by ccrawford4** [comment]
> Hello @ccrawford4 , Please download the latest version of URISanity (v0.1.5) for the fix

Thank you @isocroft 😄 Will give it a try
